### PR TITLE
chore: Add identifier to max

### PR DESCRIPTION
**Pull request by daniel-hauser**

### CODE REVIEW

File: src/scrapers/max.ts [hunk 1]
```diff
@@ -124,11 +127,11 @@ function getTransactionType(txnTypeStr: string) {
 
 function getInstallmentsInfo(comments: string) {
   if (!comments) {
-    return null;
```
Comment (esakal) [inline src/scrapers/max.ts:127]: Just wondering why you prefer to return undefined instead of null?

Comment (daniel-hauser) [reply src/scrapers/max.ts:127]: `getInstallmentsInfo` was called with `|| undefined` which is basically converts null to undefined - so it's more straight forward to just return `undefined`

**Comment by daniel-hauser** [comment]
@esakal Can you take a look at this PR?

**Comment by esakal** [comment]
@daniel-hauser I'm having technical difficulties merging it probably because it was opened before we updated the node versions. 

Two options:
1. @eshaham can you force merge it?
2. @daniel-hauser can you try to merge master into this PR

**Comment by daniel-hauser** [comment]
This branch is ahead of master.
Seems like the problem is that the node 10/12 test actions are still required by the branch protection rules - and all future PRs are blocked by it.

The fix is easy but requires access to the branch settings

**Comment by eshaham** [comment]
I thought I already did... 🤦‍♂️ 
Anyway, done now.

**Pull request merged**